### ISSUE TITLE
Show methods / members in quick documentation window

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#2039]
The plugin quick documentation currently shows something like this:

![snapshot143](https://cloud.githubusercontent.com/assets/607868/11395777/34bab0aa-936d-11e5-804c-084f7526cffc.png)

which is partially useful because the user is still left with the task to figure out the methods / members of the type being displayed (if any).

The result in the screenshot is also of type interface, which is not visible in the quick documentation (this is part of the database/sql/sql.go, https://golang.org/pkg/database/sql/#Result )
```go
// A Result summarizes an executed SQL command.
type Result interface {
	// LastInsertId returns the integer generated by the database
	// in response to a command. Typically this will be from an
	// "auto increment" column when inserting a new row. Not all
	// databases support this feature, and the syntax of such
	// statements varies.
	LastInsertId() (int64, error)

	// RowsAffected returns the number of rows affected by an
	// update, insert, or delete. Not every database or database
	// driver may support this.
	RowsAffected() (int64, error)
}
```

Also, if possible, it would be nice to have a way to open the definition of the type in the editor (useful if you navigate across several types).

**Comment by zolotov** [comment]
There is a quick definition for that. I don't see the point to add this in quickdoc

> Also, if possible, it would be nice to have a way to open the definition of the type in the editor (useful if you navigate across several types).

F4

**Comment by dlsniper** [comment]
Sure, but how can I view the quick definition for a type which I've navigated to from the quick documentation?

**Comment by zolotov** [comment]
I didn't get the question. If you have navigated to a type you've already see its definition.

**Comment by dlsniper** [comment]
Maybe this will help to clarify how what I meant: https://youtu.be/kTKIx-2ynKg

**Comment by zolotov** [comment]
Thanks, got it now. Then the question is different: did you really face the need to see interface definition in this case or it's just a thought that it would be nice to have this functionality. Because I don't see much value in this.

**Comment by dlsniper** [comment]
Thinking a bit from another perspective, I guess I'd rather see two things:
- the actual type of of the type, interface / struct
- the package path as the first row in the docs (for situations when the type returned can be part of different packages, I have this situation in one of my current projects) now that I know about the F4 tip :)

**Comment by zolotov** [comment]
Agreed. I'll include package name, import path and type of the type.

> now that I know about the F4 tip :)

There is also and edit-button in doc popup, it doesn' the same as F4

**Comment by dlsniper** [comment]
Yup, but it wasn't immediately obvious what the button does (so isn't the external doc icon, I'd rather see it as a globe or something).
I'd like to have a take on this before you, if that's ok. I think it looks simple enough to do it.

**Comment by zolotov** [comment]
> I'd like to have a take on this before you, if that's ok.

Sure, I'm not done with the quickdoc, but that's ok
